packages/panzer/disc-fe/test/la_factory/tEpetra_LOF_FilteredUGI.cpp

```diff
@@ -54,7 +54,7 @@
 #include "Thyra_SpmdVectorSpaceBase.hpp"
 
 #include "Panzer_DOFManager.hpp"
-#include "Panzer_Filtered_UniqueGlobalIndexer.hpp"
```

Comment:
Probably not worth the effort, but this file should probably be renamed to `tEpetra_LOF_FilteredGI`